meetings/2021-11-10.md

```diff
@@ -0,0 +1,192 @@
+
+        * Interesting to figure out how this would interact with spk.
+
+    * Preferred solution for Windows / macOS?
+
+        * Will need a solution for environment variables, maybe a simple source script? Aswf-docker Python package wraps Conan. In order to build packages, aswf-docker build package_name  and aswf-docker creates the right Docker image, with all the environment variables, then runs "conan create". So adding Windows support should be able to just setting environment variables, using "pip install aswf-docker", and then do "aswf-docker build", and it should just wrap the conan executable with the right environment variables. That would work for CI needs, setting environment variables isn't an issue for a CI environment. Slightly trickier for developers who just want to grab the binaries generated by ASWF, which is where we want to get rid of dependency on environment variables.
+
+* Using [linguist](https://github.com/github/linguist) in ASWF projects
+
+    * "detect blob languages, ignore binary or vendored files, suppress generated files in diffs, and generate language breakdown graphs"
+
+    * Added to aswf-docker together with Conan support
+
+    * Use in other ASWF projects? Worth promoting / documenting?
+
+    * Configured in gitattributes file, was tired of seeing auto-generated files in code reviews, one way is to .gitignore these files, but then they aren't part of the PR, so can have auto-generated files included in a PR, but yet hidden from reviews. Quite useful in this case, good to know that it exists, worth advertising, could be part of "best practices", don't know of many generated files. USD has a lot of generated files, the beginning of the file is auto-generated, but may have custom code at the end.
+
+* Need updates to [aswf-sample-project](https://github.com/AcademySoftwareFoundation/aswf-sample-project)
+
+    * GHA support, deprecate Azure Pipelines stuff?
+
+    * Test suite tool recommendations
+
+    * Conan support?
+
+* Larger GHA build instances (request from OpenVDB)
+
+    * Some can be addressed by adjusting build matrix, but may still be useful for large single builds
+
+        * Andrew: GitHub library functionality is now available, two things we can do, composable actions, which is "macro-ing" Actions together, "an Action of Actions", as well as "public workflows", available not just to your organization, but also publicly.
+
```

Comment:
Wrong links?

Reply:
Good catch, links posted by Andrew in Zoom are long gone, but I think I've found corresponding / relevant links (copy-paste between Zoom and Google Docs is always a bit iffy).